docs/en/your-profile-and-preferences/customize-group-chat-notifications.md

```diff
@@ -8,9 +8,65 @@ hide:
 
 # Customize group chat notifications
 
- :octicons-tools-24: In progress
+You can customize the messages and mentions notifications you receive in group chats. By doing so, you only receive the group chat notifications relevant to you.
 
-!!! note ""
-     We're working on this content.
+## Mute notifications in specific group chats
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-messages: **Messages**.
+    1. Long press the group chat you want to mute.
+    1. Tap :mobile-mute: **Mute**.
+    1. Select the length of time you'd like to mute your notifications.
+
+=== "Desktop"  
+
+    1. From the navigation sidebar, click :desktop-messages: **Messages**.
+    1. Under the messages list , right click the group you want to mute.
+    1. Select **Mute chat**.
+
+!!! tip
+	  Group chats show up greyed out :mobile-mute: **Mute** next to their name, when you mute notifications.
+
+## Mute group chat notifications except personal mentions
+
+If you want to avoid missing important messages, you can selectively mute some notifications, except global mentions. This procedure mutes all group chats notifications except personal mentions
```

Comment:
- 'If you want to avoid missing important messages, you can selectively mute some notifications' -> If you want to avoid missing important messages, you can mute some notifications'?
- 'This procedure mutes all group chats notifications except personal mentions' -> 'Here's how to mute all group chat notifications except personal mentions:'